web-marketplace/src/components/organisms/AgreePurchaseForm/AgreePurchaseForm.Tradable.tsx

```diff
@@ -0,0 +1,36 @@
+import Card from 'web-components/src/components/cards/Card';
+import { Body } from 'web-components/src/components/typography/Body';
+import { Title } from 'web-components/src/components/typography/Title';
+
+export type TradableProps = {
+  goToChooseCredits: () => void;
+  imgSrc: string;
+};
+export const Tradable = ({ goToChooseCredits, imgSrc }: TradableProps) => {
+  return (
+    <Card className="py-30 px-20 sm:py-50 sm:px-40 border-grey-300">
+      <Card className="p-20 border-grey-300 bg-grey-100 flex gap-10 flex-col sm:flex-row items-center sm:items-start">
+        <img
+          className="h-[100px] w-[100px]"
+          src={imgSrc}
+          alt="info with hand"
+        />
+        <div>
+          <Title className="text-base">
```

Comment:
I think this is going to be an `<h1>` without the variant prop. As there is a main heading in the page, maybe we should use a `<h2>` instead?